src/platform/apollolake/include/platform/lib/memory.h

```diff
@@ -309,7 +309,8 @@
 #define HEAP_SYSTEM_SHARED_SIZE		0x1500
 
 #define HEAP_BUFFER_BLOCK_SIZE	0x100
-#define HEAP_BUFFER_COUNT_MAX  (HP_SRAM_SIZE / HEAP_BUFFER_BLOCK_SIZE)
+/* the buffer zone will not occupy more than half of the HP SRAM on APL */
```

Comment:
Can we say here that we are enforcing this limit due to the the SRAM size limitations on APL.

Reply:
sure, updated.